### DISCUSSION TITLE
How can I plot an interaction effect between two continuous predictors?

### DISCUSSION THREAD

**Discussion by simone-anza** [FrederickHuangLin/ANCOMBC#63] [Q&A]
Hi!

Thank you for constantly developing such an amazing tool! 
I found the tutorial http://www.bioconductor.org/packages/release/bioc/vignettes/ANCOMBC/inst/doc/ANCOMBC.html very informative and useful and I have a question regarding how to plot the interaction effect between 2 continuous variables.

I would like to start from the basics. To be sure: usually, in a linear model, a negative estimate of an interaction between 2 continuous predictors means that the effect of pred1 on the response decreases when pred2 increases. Can I interpret ANCOM-BC estimates in the same way? 

I was wondering if you could give me some hints on how to proceed despite the fact that ANCOM-BC it's not a tool for visualization.

Thank you again!

Simone

**Comment by FrederickHuangLin** [comment]
Hi @simone-anza,

Yes, the interpretations you mentioned are correct.

However, adding the interaction term involved the variable of interest in ANCOMBC is generally not recommended as it changes the reference level and makes it hard to interpret the results. For details, you can check it [post](https://github.com/FrederickHuangLin/ANCOMBC/issues/58).

You can add the interaction terms for covariates other than the primary interest. 

Best,
Huang
